ocl_web/tests/ui_tests/specs/user_source_validation_spec.js

```diff
@@ -7,9 +7,8 @@ var orgPage = require('../pages/organization_page');
 var UserSourcePage = require('../pages/user_source_page');
 var ConceptPage = require('../pages/concept_page');
 var configuration = require('../utilities/configuration.js');
-var EC = require('protractor').ExpectedConditions;
 
-describe('OCL User Source Page', function () {
+describe('OCL User Source Validation Page', function () {
```

Comment:
This could remain "OCL User Source Page" "OCL User Source Edit Page", there is no "OCL User Source Validation Page"